### PR TITLE
Add ability to edit panel view cards

### LINKED ISSUE #2625: unable to edit card from UI when in panel mode

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #2625
- This PR is related to issue: 
- Link to documentation pull request: 

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/views/hui-panel-view.ts [hunk 1]
```diff
@@ -48,14 +66,31 @@ export class HUIPanelView extends UpdatingElement {
...
+    const wrapper = document.createElement("hui-card-options");
+    wrapper.hass = this.hass;
+    wrapper.lovelace = this.lovelace;
+    wrapper.path = [this.index, 0];
+    wrapper.appendChild(card);
+    this.appendChild(wrapper);
+    if (this.config!.cards!.length > 1) {
+      const warning = document.createElement("hui-warning");
+      warning.innerText =
+        "This view contains more than one card, but a panel view can only show 1 card.";
```
Comment (zsarnett) [inline src/panels/lovelace/views/hui-panel-view.ts]: Localize, yea?

**Review by zsarnett — APPROVED** [review]
Pulled and tested. Looks good!

File: src/translations/en.json [hunk 2]
```diff
@@ -2393,11 +2394,6 @@
         "tabs": {
           "info": {
             "title": "Info",
-            "remove": "Remove",
-            "set": "Set",
-            "default_ui": "{action} {name} as default page on this device",
```
Comment (zsarnett) [inline src/translations/en.json]: Just confirming these were meant to be deleted in this PR.

**Review by zsarnett — APPROVED** [review]

File: src/translations/en.json (hunk 2 above)
Comment (bramkragten) [reply src/translations/en.json]: Yeah are leftovers from States UI

**Pull request merged**